master/setup.py

```diff
@@ -164,7 +164,7 @@ def define_plugin_entries(groups):
         "buildbot.clients",
         "buildbot.data",
         "buildbot.db",
-        "buildbot.db.migrate.versions",
+        "buildbot.db.migration.versions",
```

Comment:
`buildbot.db.migration.versions` is not a python package (no `__init.py__`).

I'm not sure it should be listed here.

Reply:
I've added `__init__.py` to mark it as package.